docs/toc.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
we shouldn't be adding a toc.json anymore, right? we're doing toc.js, and it already exists

Reply:
Yeah, I'll take that out.